### PR TITLE
Specifically call out that we're not accepting new admins for now

### PR DESCRIPTION
**Pull request by tim-schilling**

### CODE REVIEW

**Review by ryancheley — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/new-admin.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: New admin
-description: Apply to be a Django Commons administrator with the following form.
+description: Apply to be a Django Commons administrator with the following form. Please note that we currently aren't accepting new admins. We will be revisiting our admin team around April of 2025. Please send your application though and we will reach out when we re-assess things.
```
Comment (Stormheg) [inline .github/ISSUE_TEMPLATE/new-admin.yml:2]: Note that GitHub calls out that this description is too long when you try to [preview the template](https://github.com/django-commons/membership/blob/7642140359290ca4fa63ad22999f609dadd8ab38/.github/ISSUE_TEMPLATE/new-admin.yml)

Screenshot 2024-10-13 at 7 17 39 AM

I don't know how significant this issue is. It appears to stop the preview from rendering - not sure if that will also happen to the live version.

The learn more link doesn't appear to go anywhere specific. I can't find information about a character limit, appears to be undocumented.

Comment (Stormheg) [reply .github/ISSUE_TEMPLATE/new-admin.yml:2]: @tim-schilling maybe we should add this callout to the markdown body instead?

```diff
-body:
-  - type: markdown
-    attributes:
-      value: |
-        Thanks for expressing interest in being an administrator for Django Commons!
+body:
+  - type: markdown
+    attributes:
+      value: |
+ Thanks for expressing interest in being an administrator for Django Commons!
+> [!NOTE]  
+> We are currently not accepting new admins. We will be revisiting our admin team around April of 2025. Please send your application though and we will reach out when we re-assess things!
```

(hope this renders as a nice callout 🤞)

Comment (tim-schilling) [reply .github/ISSUE_TEMPLATE/new-admin.yml:2]: Oh. We could a new markdown up earlier. I'll throw a few commits at it. I'm taking this as tacit approval of the messaging though :grin:

Comment (tim-schilling) [reply .github/ISSUE_TEMPLATE/new-admin.yml]: Worked beautifully! Thank you @Stormheg

Comment (Stormheg) [reply .github/ISSUE_TEMPLATE/new-admin.yml]: Amazing - thanks Tim 👍

**Pull request merged**